install_config/aggregate_logging.adoc

```diff
@@ -910,36 +898,16 @@ As part of the OpenShift cluster installation, it is recommended that you add th
 Fluentd node selector to the list of persisted
 xref:../install_config/install/advanced_install.adoc#configuring-node-host-labels[node labels].
 
-Fluentd uses `journald` as the system log source. These are log
-messages from the operating system, Docker, and OpenShift. For
-container logs, Fluentd determines which log driver Docker is using,
-`json-file` or `journald`, and automatically reads the logs from that source.
-
-// [[fluentd-use-journald]]
-// *Having Fluentd Use the Systemd Journal as the Log Source*
-
-// By default, Fluentd reads from `journald` and
-// *_/var/log/containers/<container>.log_* for system logs and container logs,
-// respectively. You can instead use the systemd journal as the log source. There
-// are three inventory parameters available:
-
-// [cols="3,7",options="header"]
-// |===
-// |Parameter
-// |Description
-
-// | `journal-source`
-// |The default is empty, so that when using the systemd journal, Fluentd first looks for
-// *_/var/log/journal_*, and if that is not available, uses *_/run/log/journal_*
-// as the journal source. You can specify `journal-source` with an explicit
-// journal path. For example, if you want Fluentd to always read logs
-// from the transient in-memory journal, set `journal-source`=*_/run/log/journal_*.
-
-// | `openshift_logging_journal_read_from_head`
-// |If this setting is `false`, Fluentd starts reading from the end of the journal,
-// ignoring historical logs. If this setting is `true`, Fluentd starts reading
-// logs from the beginning of the journal.
-// |===
+With Fluentd, it is recommended to use `json-file` as the system log source.
+These are log messages from the operating system, Docker, and OpenShift. For
+container logs, Fluentd determines which log driver Docker is using, `json-file`
+or `journald`, and automatically reads the logs from that source.
```

Comment:
Does Docker default to `json-file` now? Should we remove this line?

Reply:
Docker has always defaulted to json-file but the installer in 3.4-3.7 overwrote that default and set it to journald. Clean installs of 3.9 will use json-file as the default but environments upgraded from 3.7 will maintain their existing journald configuration.

Reply:
If we are clear that logging will use whatever Docker is configured for the logging driver, that seems sufficient, doesn't it?

Can we make this section stronger to cover what @sdodson says that since the installer does not force Docker to use a certain log driver, it will work with whatever Docker is currently using, that the user should be aware that they may have to take specific actions to change the Docker configuration to `json-file`?

Reply:
Thanks! I incorporated the explanation here and recommended that `json-file` log driver is used. (Will push the changes momentarily).